README.md

```diff
@@ -77,6 +77,7 @@ const {MsAuthPlugin} = Plugins;
 const result = await MsAuthPlugin.login({
     clientId: '<client id>',
     tenant: '<tenant, defaults to common>',
+    domainHint: '<domainHint, defaults to common>',
```

Comment:
Is it correct that the domain hint defaults to common? Or does it not default to anything?

Reply:
yes, you are right, I think default should be anything